concourse/pipeline.yml

```diff
@@ -65,34 +64,31 @@ set_failed_anchor: &set_failed
         BUCKET_PATH: clusters-google/
         BUCKET_NAME: {{tf-bucket-name}}
 #####################################################
-
 resource_types:
-- name: gcs
-  type: docker-image
-  source:
-    repository: frodenas/gcs-resource
-
-- name: terraform
-  type: docker-image
-  source:
-    repository: ljfranklin/terraform-resource
-    tag: 0.11.14
+    - name: terraform
+      type: registry-image
+      source:
+        repository: ljfranklin/terraform-resource
+        tag: 0.11.14
+    - name: gcs
+      type: registry-image
+      source:
+        repository: frodenas/gcs-resource
 
 resources:
 # Image Resources
 - name: centos-gpdb-dev-7
   type: docker-image
   source:
-    repository: pivotaldata/centos-gpdb-dev
-    tag: '7-gcc6.2-llvm3.7'
+    repository: gcr.io/data-gpdb-public-images/gpdb5-centos7-build-test
+    tag: latest
 
 - name: ubuntu18-image-test
   type: docker-image
```

Comment:
and here
just run `sed -i s/docker-image/registry-image/g`

Reply:
Cool, learned that~